src/services/recim/match.ts

```diff
@@ -123,6 +123,13 @@ const updateMatch = (ID: number, updatedMatch: PatchMatch): Promise<CreatedMatch
 const deleteMatchCascade = async (matchID: number): Promise<CreatedMatch> =>
   http.del(`recim/matches/${matchID}`);
 
+// shortcut service to delete a list of matches
+const deleteMatchList = async (matches: Match[]) => {
```

Comment:
Truthfully, I don't understand why we would have this. Can't we just loop in whatever function we want to delete the match with?

Reply:
It's simply a shortcut for a loop I can foresee being used in other place in the future:
- delete all activity matches
- admin page delete matches

This is definitely future proofing so potentially I could remove it but I can already see myself adding it back in within the next month or two.